amity.py

```diff
@@ -662,21 +505,22 @@ def list_living_spaces(self):
 
```

Comment:
Try combining the functions list_offices and list_living_spaces in one function that will serve the purpose of displaying all rooms and its occupants, maybe in functions print_unallocated and print_allocations.